common/src/chain/tokens/issuance.rs

```diff
@@ -35,6 +35,22 @@ pub enum TokenTotalSupply {
     Unlimited, // limited only by the Amount data type
 }
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Encode, Decode, serde::Serialize)]
+pub enum IsTokenFreezable {
+    #[codec(index = 0)]
+    Yes,
```

Comment:
Can we make No=0, and Yes=1, as in false/true casting to int? This is very confusing. I made this comment about the wallet, but it seems I was wrong and it's in chainstate.